packages/kbn-scout/src/playwright/config/create_config.test.ts

```diff
@@ -7,26 +7,35 @@
  * License v3.0 only", or the "Server Side Public License, v 1".
  */
 
-import { SCOUT_REPORTER_ENABLED, SCOUT_SERVERS_ROOT } from '@kbn/scout-info';
+import { SCOUT_SERVERS_ROOT } from '@kbn/scout-info';
+import { scoutPlaywrightReporter, scoutFailedTestsReporter } from '@kbn/scout-reporting';
 import { createPlaywrightConfig } from './create_config';
 import { VALID_CONFIG_MARKER } from '../types';
 import { generateTestRunId } from '@kbn/scout-reporting';
 
 jest.mock('@kbn/scout-reporting', () => ({
   ...jest.requireActual('@kbn/scout-reporting'),
   generateTestRunId: jest.fn(),
+  scoutPlaywrightReporter: jest.fn(),
+  scoutFailedTestsReporter: jest.fn(),
 }));
 
 describe('createPlaywrightConfig', () => {
+  const mockedRunId = 'mocked-run-id';
   const mockGenerateTestRunId = generateTestRunId as jest.Mock;
+  const mockedScoutPlaywrightReporter = scoutPlaywrightReporter as jest.Mock;
+  const mockedScoutFailedTestsReporter = scoutFailedTestsReporter as jest.Mock;
 
   beforeEach(() => {
     jest.clearAllMocks();
+    delete process.env.TEST_RUN_ID;
```

Comment:
I had no idea you could do this...deleting an env var that was on the cli?  Pretty cool

Reply:
I found few examples of it online: not sure if it is the most optimal, but it works